src/jdk.jfr/share/classes/jdk/jfr/internal/LogTag.java

```diff
@@ -97,7 +97,11 @@ public enum LogTag {
     /**
      * -XX:StartFlightRecording
      */
-    JFR_START(15);
+    JFR_START(15),
+    /**
+     *  Covers usage of MethodTiming and MethodTrace events
+     */
+    JFR_METHODTRACE(16);
```

Comment:
You can consider adding a trailing `,` and moving `;` onto a new line if addition to this enum is frequent.